targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_WIO_BG96/onboard_modem_api.c

```diff
@@ -0,0 +1,70 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017 ARM Limited
```

Comment:
Can you fix the year (in all new files in this PR), should be 2018

we should also be using SDPX identifier (suggested to be used in our license documentation)

```

 * Copyright (c) 2018 ARM Limited
 * SPDX-License-Identifier: Apache-2.0

Reply:
Yes, correct, thanks for correction